src/actions/deleteEmptyThought.ts

```diff
@@ -147,7 +147,11 @@ export const deleteEmptyThoughtActionCreator: Thunk = (dispatch, getState) => {
   const uneditable = contextOfCursor && findDescendant(state, head(cursor), '=uneditable')
 
   if (prevThought && uneditable) {
-    dispatch(error({ value: `'${ellipsize(headValue(state, cursor))}' is uneditable and cannot be merged.` }))
+    dispatch(
+      error({
+        value: `'${ellipsize(headValue(state, cursor) ?? 'MISSING_THOUGHT')}' is uneditable and cannot be merged.`,
```

Comment:
i dunno, feels like these cases could be just `!`-ed, if we don't want to give some information token like `~missing/empty/??~/

- i don't want to create a variable for this
- fallbacking like this in ellipsize, and making it go through it also is an eye-sore, but i dont want to extract a variable, again
- I also thought maybe an `ellipsize` could take `str | undefined`, but it sounds so evil

so wdyt? :D